Library/Formula/openssl.rb

```diff
@@ -21,12 +21,20 @@ class Openssl < Formula
   keg_only :provided_by_osx,
     "Apple has deprecated use of OpenSSL in favor of its own TLS and crypto libraries"
 
-  # This is a workaround for Apple removing the Equifax Secure CA root from the System in 10.10.3
-  # Their doing so has broken certificate verification and consquently secure connection for dependants.
-  # Scope this to Yosemite and remove immediately once Apple have fixed the issue.
-  resource "Equifax_CA" do
-    url "https://www.geotrust.com/resources/root_certificates/certificates/Equifax_Secure_Certificate_Authority.pem"
-    sha256 "f24e19fb93983b4fd0a377335613305f330c699892c789356eb216449804d0e9"
+  # Remove both patches with the 1.0.2b release.
+  # They fix:
+  # https://github.com/Homebrew/homebrew/pull/38495
+  # https://github.com/Homebrew/homebrew/issues/38491
+  # Upstream discussions:
+  # https://www.mail-archive.com/openssl-dev@openssl.org/msg38674.html
+  patch do
+    url "https://github.com/openssl/openssl/commit/6281abc796234.diff"
```

Comment:
We don't use ` .patch ` because the ` git ` version is appended and it can break if upgraded or something like that. @mikemcquaid explains it more eloquently than I do.